pkg/edit/builtins_test.go

```diff
@@ -208,6 +208,71 @@ var bufferBuiltinsTests = []struct {
 		tk.CodeBuffer{Content: "ab", Dot: 1},
 		tk.CodeBuffer{Content: "a", Dot: 1},
 	},
+	{
+		"transpose-char",
+		tk.CodeBuffer{Content: "ab", Dot: 0},
+		tk.CodeBuffer{Content: "ba", Dot: 2},
+	},
+	{
+		"transpose-char",
+		tk.CodeBuffer{Content: "ab", Dot: 1},
+		tk.CodeBuffer{Content: "ba", Dot: 2},
+	},
+	{
+		"transpose-char",
+		tk.CodeBuffer{Content: "ab", Dot: 2},
+		tk.CodeBuffer{Content: "ba", Dot: 2},
+	},
+	{
+		"transpose-char",
+		tk.CodeBuffer{Content: "abc", Dot: 1},
+		tk.CodeBuffer{Content: "bac", Dot: 2},
+	},
+	{
+		"transpose-char",
+		tk.CodeBuffer{Content: "a", Dot: 1},
+		tk.CodeBuffer{Content: "a", Dot: 1},
+	},
+	{
+		"transpose-char",
+		tk.CodeBuffer{Content: "a", Dot: 0},
+		tk.CodeBuffer{Content: "a", Dot: 0},
+	},
+	{
+		"transpose-char",
+		tk.CodeBuffer{Content: "a", Dot: 0},
+		tk.CodeBuffer{Content: "a", Dot: 0},
+	},
+	{
+		"transpose-char",
+		tk.CodeBuffer{Content: "a", Dot: 0},
+		tk.CodeBuffer{Content: "a", Dot: 0},
+	},
+	{
+		"transpose-word",
+		tk.CodeBuffer{Content: "ab  bc cd", Dot: 0},
+		tk.CodeBuffer{Content: "bc  ab cd", Dot: 6},
+	},
+	{
+		"transpose-word",
+		tk.CodeBuffer{Content: "ab  bc cd", Dot: 6},
+		tk.CodeBuffer{Content: "ab  cd bc", Dot: 9},
+	},
+	{
+		"transpose-word",
+		tk.CodeBuffer{Content: " ab  ", Dot: 4},
+		tk.CodeBuffer{Content: " ab  ", Dot: 4},
+	},
+	{
+		"transpose-word",
+		tk.CodeBuffer{Content: " \t\n  ", Dot: 4},
+		tk.CodeBuffer{Content: " \t\n  ", Dot: 4},
+	},
+	{
+		"transpose-word",
+		tk.CodeBuffer{Content: "ab  bc cd", Dot: 9},
+		tk.CodeBuffer{Content: "ab  cd bc", Dot: 9},
```

Comment:
Please add at least one test case for `transpose-small-word` and `transpose-alnum-word`.